rts/lib/lua/CMakeLists.txt

```diff
@@ -49,11 +48,11 @@ else ()
 TARGET_LINK_LIBRARIES(lua)
 endif ()
 
+target_link_libraries(lua GLEW::GLEW)
```

Comment:
is it actually needed? I don't see glew referenced in LuaUser.cpp.

Reply:
Yeah looks like it's needed in `LuaShaders.hpp` it includes `Rendering/GL/myGL.h` which includes `GLEW`